roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -260,6 +261,16 @@ def get_var(self, *keys, **kwargs):
                 )
             value = kwargs["default"]
 
+        if isinstance(value, string_types) and self._templar:
```

Comment:
I don't like this implementation.

If I retrieve an item that is a dictionary, and that dictionary has a key that needs to be templated, this is not going to template those keys for me.

I think we should leave the templar function in-line for the variables that need it until we can come up with something more robust.

Reply:
If a value is a composite, would other ansible actions/tasks be templating the contents? I don't think so. And if code needs to do that, it can still do it; this doesn't stand in the way.

That case excepted, this makes it so that you do not have to wait for someone to report that "I tried to put a variable in there and it failed" with the accompanying debugging/confusion. It's already done. Is that not an improvement?